internal/handlers/user_test.go

```diff
@@ -1047,6 +1047,160 @@ func (s *UserTestSuite) TestUser_DeleteUserFunc_OK() {
 	s.userRepository.AssertExpectations(t)
 }
 
+func (s *UserTestSuite) TestUser_DeleteUser_OK() {
+	t := s.T()
+
+	userID := 1232
```

Comment:
I think that the usage "mock.Anything" is unnecessary because context can be used instead of it.
```go
        ctx := context.Background()
	request := http.Request{}
	data := users.DeleteUserParams{
		HTTPRequest: request.WithContext(ctx),
		UserID:      int64(userID),
	}
	expectedUser := &ent.User{IsReadonly: true}

	s.userRepository.On("GetUserByID", ctx, userID).Return(expectedUser, nil)
	s.userRepository.On("Delete", ctx, userID).Return(nil)
```

Reply:
Done. Thanks for the feedback!

Reply:
Sorry, but I don't see any changes.

Reply:
Sorry, I haven't pushed all changes.